### PR TITLE
Gateway : Bridge to Alt L1s

### LINKED ISSUE #296: Supports multi-chain bridges in gateway
* On the Ethereum Wallet page, we need a new button next to `Fast Bridge to L2` for Boba Token.
image
* The popup model should be
image
* There should be another tab called `Bridge between L1s` in the history. 
image
@trangtran-enya will provide an endpoint for you to pull data.

### PR DESCRIPTION
**Pull request by sk-enya**
:clipboard:  closes: https://github.com/bobanetwork/boba/issues/296
## Overview
Please check the issues attached above for more details.

## Changes

Markup changes for 
1. wallet page 
2. multichain input 
3. history page.

## Testing
TBD.

UI
Screenshot 2022-08-24 at 8 38 07 PM
Screenshot 2022-08-24 at 8 38 11 PM

### CODE REVIEW

**Comment by boyuan-chen** [comment]
Could you please add a dropdown for the target L1 chain?

**Comment by sk-enya** [comment]
> Could you please add a dropdown for the target L1 chain?

Sure, Can you please help me what will be the options in dropdown?

**Comment by boyuan-chen** [comment]
> > Could you please add a dropdown for the target L1 chain?
> 
> Sure, Can you please help me what will be the options in dropdown?

BNB, Avalanche, Fantom, Moonbeam

**Comment by sk-enya** [comment]
@Boyuan-Chen currently BNB, Fantom, Avalanche Is implemented.
Can you please help me with more details on how to get support for moonbase added?

**Comment by boyuan-chen** [comment]
> @Boyuan-Chen currently BNB, Fantom, Avalanche Is implemented.
> Can you please help me with more details on how to get support for moonbase added?

Layer zero didn't deploy their protocol on moonbase.

**Comment by sk-enya** [comment]
@Boyuan-Chen now only history page is pending with needed api details for alt l1 txs

**Comment by boyuan-chen** [comment]
> @Boyuan-Chen now only history page is pending with needed api details for alt l1 txs

Yup! @trangtran-enya The frontend is ready.

**Review by boyuan-chen — APPROVED** [review]

**Review by InoMurko — APPROVED** [review]

**Review by InoMurko — APPROVED** [review]
LGTM

File: packages/boba/gateway/src/services/networkService.js [hunk 1]
```diff
@@ -1262,7 +1262,7 @@ class NetworkService {
     ).post('get.layerzero.transactions', {
       address: this.account,
       fromRange: 0,
-      toRange: 12418344,
+      toRange: 1000,
```
Comment (InoMurko) [inline packages/boba/gateway/src/services/networkService.js:1265]: this could be cached if we wouldn't be using POST for everything :-)

**Comment by sk-enya** [comment]
@Boyuan-Chen can you please review this PR once everything seems good to me here now.

**Review by boyuan-chen — APPROVED** [review]
LGTM

**Pull request merged**